api-java-mixin.raml

```diff
@@ -644,47 +996,18 @@ types:
           instance.setPreciseAmount(template.getPreciseAmount());
           return instance;
       }
-      @Override
-      default javax.money.MonetaryOperator createMoneyOperator() {
```

Comment:
Please revert this as these methods override the createMoneyOperator from the Money class. Also it seems to work this could lead to confusion because of the override directive